packages/mediapipe-task-text/test/text_classifier_result_test.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:ffi';
+import 'package:ffi/ffi.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mediapipe_core/mediapipe_core.dart';
+import 'package:mediapipe_text/mediapipe_text.dart';
+import 'package:mediapipe_core/src/third_party/mediapipe/generated/mediapipe_common_bindings.dart'
+    as core_bindings;
+import 'package:mediapipe_text/src/third_party/mediapipe/generated/mediapipe_text_bindings.dart'
+    as bindings;
+
+void main() {
+  group('TextClassifierResult.structToDart should', () {
+    // test('load an empty object', () {
```

Comment:
Should this test be enabled?

Reply:
it is restored! (and I have no recollection of why it was ever commented out - possibly only to reduce noise from around the failing test I was debugging 🤷 )